pages/common/ccache.md

```diff
@@ -0,0 +1,24 @@
+# ccache
+
+> C/C++ compiler cache.
+> More information: <https://ccache.dev/manual/latest.html>.
```

Comment:
How about mentioning it in the description instead?
```suggestion
> C/C++ compiler cache.
> Note: packages usually provide symlinks to compilers in `/usr/lib/ccache/bin`. Prepend this directory to `$PATH` to use them.
> More information: <https://ccache.dev/manual/latest.html>.
```

Reply:
Sure ! I pushed a new commit to do that (but modified it a bit)